src/poetry/core/version/pep440/segments.py

```diff
@@ -102,6 +103,53 @@ def next_patch(self) -> Release:
             extra=tuple(0 for _ in self.extra),
         )
 
+    def next(self) -> Release:
```

Comment:
Same here, I'd like some explicit tests. Seems like tests for Release are in `test_version_pep440.py`.